_posts/2020-04-20-schedule-your-functions.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "On time - schedule your OpenFaaS functions"
+description: "In this post Martin will show you how to deploy a component which will execute your functions on a schedule"
```

Comment:
"In this post Martin will show you how schedule your functions with the cron-connector, for regular tasks and scheduled jobs"

Reply:
(Adding the why)

Reply:
Sounds better, changed :heavy_check_mark: